core/tests/org.fusesource.ide.camel.model.service.core.tests/src/test/java/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactoryTest.java

```diff
@@ -27,6 +27,12 @@
 																		 "2.15.1",
 																		 "2.17.0.redhat-630077", 
 																		 "2.17.0.redhat-630159");
+	@Test
+	public void testGetLatestCamelVersion() {
```

Comment:
camel model factory are initialized by extension points, so this test needs to be an integration one.